packages/react-wallet/src/contexts/WalletContext.ts

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { FullWallet, ProviderType, WalletService } from '../../../wallet/dist';
```

Comment:
```suggestion
import { FullWallet, ProviderType, WalletService } from '@sovryn/wallet';
```
Packages will not compile for publishing otherwise